utils/ccloud_library.sh

```diff
@@ -549,6 +549,41 @@ function ccloud::delete_acls_ccloud_stack() {
   return 0
 }
 
+
+# ccloud::set_cli_from_config_file enables users to have multiple properties files
+# generated by ccloud-stack and switch between them
+function ccloud::set_cli_from_config_file() {
+  [ -z "$1" ] && {
+    echo "ccloud::validate_ccloud_config expects one parameter (configuration file with Confluent Cloud connection information)"
+    exit 1
+  }
+
+  local cfg_file="$1"
+  ccloud::validate_logged_in_cli
+  ccloud::validate_ccloud_config $cfg_file
+  ccloud::generate_configs $cfg_file
+  source delta_configs/env.delta
+
+  if [[ ! -z "$ENVIRONMENT_ID" && "$ENVIRONMENT_ID" != -1 ]] ; then
+    echo "confluent environment use $ENVIRONMENT_ID"
+    confluent environment use $ENVIRONMENT_ID
+
+    if [[ ! -z "$KAFKA_CLUSTER_ID" && "$KAFKA_CLUSTER_ID" != -1 ]] ; then
+      echo "confluent kafka cluster use $KAFKA_CLUSTER_ID"
+      confluent kafka cluster use $KAFKA_CLUSTER_ID
+    else
+      echo "Cannot set kafka cluster from $KAFKA_CLUSTER_ID"
```

Comment:
Judging from the condition, the ID could be `""`, so we should put quotes around it for clarity.
```suggestion
      echo "Cannot set kafka cluster from \"$KAFKA_CLUSTER_ID\""
```